### ISSUE TITLE
Import older NEM data and match DUIDs

### ISSUE THREAD

**Issue by nc9** [opennem/opennem#105]
Currently the NEM start dates for data are:

```
price_first_seen=datetime.fromisoformat("2009-07-01T00:00:00+10:00"),
interconnector_first_seen=datetime.fromisoformat("2010-01-01T00:00:00+10:00")
```

There is `dispatch_price` and interconnector data for prior to both of these dates available that needs to be imported

note: on early `wind` data for `NEM`

**Comment by nc9** [comment]
looks like a bit of non-scheduled missing in < ~2010

**Comment by nc9** [comment]
merged into #190
